packages/ckeditor5-package-generator/lib/utils/validate-global-name.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license Copyright (c) 2020-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+
+/**
+ * If the global name is not valid, prints the error.
+ *
+ * @param {Logger} logger
+ * @param {String} globalName
+ */
+module.exports = function validateGlobalName( logger, globalName ) {
+	if ( !globalName.length ) {
+		return;
```

Comment:
I think that would be good also to log some info that `Global name cannot be an empty string.` when someone hits `Enter` when input prompt comes.

WDYT @psmyrek , @przemyslaw-zan ?

Reply:
I think that `inquirer` should have an elegant way of disallowing empty input.

Reply:
Addressed in a9c1db6.